Client/Frontend/Home/MessageCard/HomepageMessageCardViewModel.swift

```diff
@@ -98,6 +98,7 @@ extension HomepageMessageCardViewModel: HomepageViewModelProtocol {
     }
 
     func refreshData(for traitCollection: UITraitCollection,
+                     size: CGSize,
                      isPortrait: Bool = UIWindow.isPortrait,
                      device: UIUserInterfaceIdiom = UIDevice.current.userInterfaceIdiom) {}
```

Comment:
I think this could be removed since we have a default already for that in the HomepageViewModelProtocol, as it was done for other view models